### ISSUE TITLE
Add SlimerJS 0.9.5 to Travis CI testing

### ISSUE THREAD

**Issue by paazmaya** [n1k0/casperjs#1141]
Since the PhantomJS was just updated to `1.9.8`, noticed that the latest SlimerJS tested is `0.8.5`.

> SlimerJS 0.9.5 has been released on January 25th, 2015. 

http://docs.slimerjs.org/0.9/release-notes.html

In case it is as trivial as adding the snippet below, I am up for a PR:

```yml
- ENGINE="slimerjs" ENGINE_VERSION="0.9.5" MAKE_TEST_COMMAND="test-dotNET"
ENGINE_ARCHIVE_URL="http://download.slimerjs.org/releases/0.9.5/slimerjs-0.9.5-linux-x86_64.tar.bz2"
- ENGINE="slimerjs" ENGINE_VERSION="0.9.5" MAKE_TEST_COMMAND="test"
ENGINE_ARCHIVE_URL="http://download.slimerjs.org/releases/0.9.5/slimerjs-0.9.5-linux-x86_64.tar.bz2"
```

Also it seems that the existing URLs are invalid. Now all versions are under
http://download.slimerjs.org/releases/

**Comment by n1k0** [comment]
Could you make the PR for this?

**Comment by paazmaya** [comment]
Yes sir!

**Comment by paazmaya** [comment]
Merged in e20682676dfdd90c27af268d4d50c7610ee67fa3
